layers/meta-resin-raspberrypi/recipes-connectivity/bluez5/bluez5_%.bbappend

```diff
@@ -18,6 +18,13 @@ pkg_postinst_${PN}_append_raspberrypi () {
 	systemctl $OPTS mask brcm43438.service
 }
 
+SRC_URI_remove_raspberrypi3-64 = " \
```

Comment:
It depends on which of the 4 boards finished first. If any of rpi, rpi2 or rpi3 (and not rpi3 64 bits) was built first, the first PR passed